src/uu/install/src/install.rs

```diff
@@ -164,17 +168,15 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
                 .help("apply access/modification times of SOURCE files to corresponding destination files")
         )
         .arg(
-            // TODO implement flag
             Arg::with_name(OPT_STRIP)
             .short("s")
             .long(OPT_STRIP)
-            .help("(unimplemented) strip symbol tables")
+            .help("strip symbol tables")
```

Comment:
you should add on which os it is supported
example:
```suggestion
            .help("strip symbol tables (no action Windows)")
```